source/sections/_lose_the_spreadsheet.html.erb

```diff
@@ -21,6 +21,10 @@
             <%= image_tag "icons/#{col.icon}.svg", class: "icon" %>
             <h3 class="heading"><%= col.title %></h3>
             <p><%= col.description %></p>
+            <br />
+            <a href="<%= col.link %>" title="Learn More" class="featured_link__learn-more btn--solid btn--secondary">
```

Comment:
I wonder if there's a better way to do these links, upon further review.

1. `col.link` actually just does `href="/dividends"`, but it should be `href="/nz/dividends"`, etc.
2. Luckily, our Javascript does localize this for you, replacing `/dividends` with `/nz/dividends`—however, this may not happen for SEO.

I believe we can do this no problem if you have a `locale_obj`, which we should have (or `current_locale_obj` as other places use):
```
<%= localize_path(col.link, locale_id: locals[:locale_obj][:id]) %>
```

Reply:
Good spot!